uwsgi.ini

```diff
@@ -25,7 +25,7 @@ pidfile=/tmp/micromasters-mast.pid
 vacuum=True
 enable-threads = true
 single-interpreter = true
-static-map = /static=./static
+static-map = /static=./staticfiles
```

Comment:
I don't think this is necessarily the proper fix. We can actually remove this line entirely now that Nginx is serving the static assets. The static-map is only a best effort setting. http://uwsgi-docs.readthedocs.io/en/latest/StaticFiles.html#mode-3-using-static-file-mount-points

In the current configuration this shouldn't even be doing anything.